Makefile

```
@@ -283,5 +327,5 @@ test: capella/readonly t4c/client/remote
 		docker push "$(DOCKER_REGISTRY)/$(DOCKER_PREFIX)$(IMAGENAME):$$DOCKER_TAG";\
 	fi
 
-.PHONY: *
+.PHONY: tests/* *
```

```suggestion
.PHONY: tests/* t4c/* *
```

The server images are not build on my side, "already up to date"